wayland 1.2.1 undefined reference

With wayland 1.2.1 development headers, the build fails with the following errors:

In function `wlf_DestroyInput':
undefined reference to `wl_pointer_release'
undefined reference to `wl_keyboard_release'

I do not intend to use the wayland client, it just turns out I have the development headers installed on my machine and the wayland client got automatically built. Quick googling shows the correct functions may be wl_pointer_destroy and wl_keyboard_destroy, but I do not know in which version the change took place.

--- a/cmake/FindWayland.cmake
+++ b/cmake/FindWayland.cmake
@@ -28,7 +28,7 @@
 # limitations under the License.
 #=============================================================================
 
-set(REQUIRED_WAYLAND_CLIENT_VERSION 0.95)
+set(REQUIRED_WAYLAND_CLIENT_VERSION 1.3.0)
 include(FindPkgConfig)
 
 if(PKG_CONFIG_FOUND)

